docs/concepts/jcloud/yaml-spec.md

```diff
@@ -69,83 +81,27 @@ If GPU is enabled, JCloud will provide NVIDIA A10G Tensor Core GPUs with 24G mem
 When using GPU resources, it may take a few extra minutes before all Executors are ready to serve traffic.
 ```
 
-#### Shared GPU
+| Instance | GPU    | Memory | Credit Per Hour |
+|----------|--------|--------|-----------------|
+| G1       | shared | 14G    | 100             |
+| G2       | 1      | 14G    | 125             |
+| G3       | 2      | 24G    | 250             |
+| G4       | 4      | 56G    | 500             |
+
+##### Shared GPU
 
 An Executor using a `shared` GPU shares this GPU with up to four other Executors.
 This enables time-slicing, which allows workloads that land on oversubscribed GPUs to interleave with one another.
 
-```{code-block} yaml
----
-emphasize-lines: 5-7
----
-jtype: Flow
-executors:
-  - name: executor1
-    uses: jinaai+docker://<username>/Executor1
-    jcloud:
-      resources:
-        gpu: shared
-```
-
-```{caution}
 The tradeoffs with a `shared` GPU are increased latency, jitter, and potential out-of-memory (OOM) conditions when many different applications are time-slicing on the GPU. If your application is memory consuming, we suggest using a dedicated GPU.
-```
 
-#### Dedicated GPU
+To use `shared` GPU, `G1` needs to be specificied as Instance Type.
```

Comment:
```suggestion
To use `shared` GPU, `G1` needs to be specificied as Instance Type.
```
Why are we capitalizing `Instance Type`?

Reply:
I just thought it's a special term we come up with, or you think it should be`Instance type` or just all lower cased `instance type`?

Reply:
Which of us came up with it? If no compelling reason otherwise, I'd go with `instance type` (all lower case)

Reply:
I did, yea no compelling reason, switched to all lower case.